### ISSUE TITLE
SHACL-Webinar: Integrated Background Knowledge & Validating of External Classes

### ISSUE THREAD

**Issue by init-dcat-ap-de** [SEMICeu/DCAT-AP#163]
Thank you again for doing the SHACL-Webinar and releasing the [screencast](https://www.youtube.com/watch?v=9I8IOhsg2hg). We now had a chance to revisit the discussed problems and solutions. (See: https://github.com/SEMICeu/DCAT-AP/issues/162)
We want to move forward with the validation of DCAT-AP.de for GovData.de and we would like to do so by re-using the shapes from DCAT-AP.

The first discussion was: Which background knowledge should be available for the shapes? We propose **foaf** and **vcard**. 
Those classes will be mostly included within a catalogue and they are widely known and used. So it’s highly likely, that a `dct:publisher`will be provided as an `foaf:Organization` without being also declared as an `foaf:Agent`.

The second part of this discussion was: Which classes should be checked by SHACL-rules?
Here we propose the following classes. Our idea was again to check what’s usually within the file, at minimum the core-classes of DCAT:
* dcat:Dataset
* dcat:Catalog
* dcat:CatalogRecord
* dcat:Distribution

Highly recommended:
* foaf:Agent
* vcard:Kind

Open for discussion:
* dct:Location
* dct:Frequency
* adms:Identifier
* dct:PeriodOfTime
* spdc:Checksum

 
![grafik](https://user-images.githubusercontent.com/14979304/95754323-254df280-0ca3-11eb-8fbb-aa39852f03b0.png)

What do you think? 
Part of this discussion has to be how we announce, which background knowledge is available. Adding the RDFs to the configuration of the validator is one way, it should also be possible to use an owl:import within the SHACL-Shapes.

The EDP-Metadata-Quality dashboard shows 0% DCAT-AP compliance for almost everyone. That should improve with better SHACL-Shapes.

**Comment by init-dcat-ap-de** [comment]
Related issues: #115, #116, #117, #118, #125

**Comment by bertvannuffelen** [comment]
In the branch a concrete proposal https://github.com/SEMICeu/DCAT-AP/tree/2.1.0-draft/releases/2.1.0 on the imports has been made. The import file for the vocabulary contains almost all toplevel imports of the used vocabularies.

**Comment by init-dcat-ap-de** [comment]
You include https://www.w3.org/ns/dcat2.ttl.
This results in Catalogs being a subclass of Datasets. That is not true for DCAT-AP 2.0, as far as I can see.

It doesn't seem to interfere with your SHACL-shapes, but we got some errors (since DCAT-AP.de has additional properties for Datasets but not for Catalogs).

**Comment by init-dcat-ap-de** [comment]
You did not include http://www.w3.org/ns/org.ttl

But according to the specification: "If the Agent is an organisation, the use of the Organization Ontology is recommended." (page 10).

**Comment by bertvannuffelen** [comment]
Can you give an example where it goes wrong? The issue is that W3C has declared Catalog a sublcassof Dataset and that is something a profile cannot negate. If that leads to problems we should have W3C revert that.

> 
> 
> You include https://www.w3.org/ns/dcat2.ttl.
> This results in Catalogs being a subclass of Datasets. That is not true for DCAT-AP 2.0, as far as I can see.
> 
> It doesn't seem to interfere with your SHACL-shapes, but we got some errors (since DCAT-AP.de has additional properties for Datasets but not for Catalogs).

**Comment by bertvannuffelen** [comment]
> 
> 
> You did not include http://www.w3.org/ns/org.ttl
> 
> But according to the specification, it is needed: "If the Agent is an organisation, the use of the Organization Ontology is recommended." (page 10).

Good point, this is a nice one. But how does that influences the SHACL contraints? It is hard to decide if a publisher has expresses an agent with name "European Commission" to derive with a machine that this agent should be described according to the registered organisations vocabulary.

**Comment by bertvannuffelen** [comment]
Post it as a question to w3c https://github.com/w3c/dxwg/issues/1387
